src/measure_theory/integral/riesz_markov_kakutani.lean

```diff
@@ -109,3 +112,140 @@ begin
+end
+end urysohn_bounded_nnreal
+
+/-- A positive linear functional on bounded continuous functions on X is monotone. -/
+lemma positive_linear_functional_mono {f g : X →ᵇ ℝ≥0} (f_le_g : ∀ (x : X), f x ≤ g x) :
+  Λ f ≤ Λ g :=
+begin
+  have g_decomposition : g = (g.nnreal_sub f) + f,
+  { ext x,
+    rw [bounded_continuous_function.coe_add, pi.add_apply,
+      bounded_continuous_function.nnreal_sub_apply, nonneg.coe_add, nnreal.coe_sub (f_le_g x)],
+    ring, },
+  rw g_decomposition,
+  simp only [map_add, le_add_iff_nonneg_left, zero_le'],
+end
+
+variables [compact_space X] [t2_space X]
+
+/-- The Riesz content (associated to a given positive linear functional Λ) is
+finitely additive: λ(K₁ ∪ K₂) = λ(K₁) + λ(K₂) for any disjoint compact subsets K₁, K₂ ⊆ X. -/
+lemma riesz_content_additive {K₁ K₂ : compacts X} (disj : disjoint K₁ K₂) :
+  riesz_content_aux Λ (K₁ ⊔ K₂) = riesz_content_aux Λ K₁ + riesz_content_aux Λ K₂ :=
+begin
+  refine le_antisymm (riesz_content_aux_sup_le Λ K₁ K₂) _,
+  refine le_cInf (riesz_content_aux_image_nonempty Λ (K₁ ⊔ K₂)) _,
+  rintros b ⟨f, ⟨hf, Λf_eq_b⟩⟩,
+  haveI : normal_space X := normal_of_compact_t2,
+  obtain ⟨g₁, g₂, ⟨hg₁, hg₂, h_sum⟩⟩ :=
+    exists_sum_one_of_closed_nnreal K₁.is_compact.is_closed K₂.is_compact.is_closed disj,
+  have f_eq_sum : f = g₁.nnreal_mul f + g₂.nnreal_mul f,
+  { ext x,
+    simp only [bounded_continuous_function.coe_add, pi.add_apply,
+               bounded_continuous_function.nnreal_mul_apply, nonneg.coe_add, nonneg.coe_mul],
+    rw [← nnreal.coe_mul, ← nnreal.coe_mul, ← nnreal.coe_add, ← add_mul,
+        (show g₁ x + g₂ x = (g₁ + g₂) x, by refl), h_sum],
+    simp only [bounded_continuous_function.coe_one, pi.one_apply, one_mul], },
+  rw [← Λf_eq_b, f_eq_sum, map_add],
+  have aux₁ : ∀ x ∈ K₁, 1 ≤ (g₁.nnreal_mul f) x,
+  { intros x hx,
+    simp only [hg₁ hx, pi.one_apply, bounded_continuous_function.nnreal_mul_apply, one_mul],
+    exact hf x (mem_union_left _ hx), },
+  have aux₂ : ∀ x ∈ K₂, 1 ≤ (g₂.nnreal_mul f) x,
+  { intros x hx,
+    simp only [hg₂ hx, pi.one_apply, bounded_continuous_function.nnreal_mul_apply, one_mul],
+    exact hf x (mem_union_right _ hx), },
+  apply add_le_add (riesz_content_aux_le Λ aux₁)
+                   (riesz_content_aux_le Λ aux₂),
+end
+
+section riesz_content
+/-- Given a positive linear functional Λ on a compact Hausdorff space X, a content λ is
+defined on compact subsets K ⊆ X by λ(K) = inf {Λf | 1≤f on K}. -/
+def riesz_content (Λ : (X →ᵇ ℝ≥0) →ₗ[ℝ≥0] ℝ≥0) : content X :=
+{ to_fun := riesz_content_aux Λ,
+  mono' := λ K₁ K₂, riesz_content_aux_mono Λ,
+  sup_disjoint' := λ K₁ K₂, riesz_content_additive Λ,
+  sup_le' := riesz_content_aux_sup_le Λ, }
+
+lemma riesz_content.to_fun_eq (Λ : (X →ᵇ ℝ≥0) →ₗ[ℝ≥0] ℝ≥0) :
+  (riesz_content Λ).to_fun = riesz_content_aux Λ := rfl
+
+lemma riesz_content.coe_to_fun_eq (Λ : (X →ᵇ ℝ≥0) →ₗ[ℝ≥0] ℝ≥0) :
+  ⇑(riesz_content Λ) = (coe : ℝ≥0 → ℝ≥0∞) ∘ (riesz_content_aux Λ) := rfl
```

Comment:
```suggestion
@[simp, norm_cast] lemma coe_riesz_content (Λ : (X →ᵇ ℝ≥0) →ₗ[ℝ≥0] ℝ≥0) :
  ⇑(riesz_content Λ) = (coe : ℝ≥0 → ℝ≥0∞) ∘ (riesz_content_aux Λ) := rfl
```
would be the way to write it. Or even
```suggestion
@[simp, norm_cast] lemma riesz_content_apply (Λ : (X →ᵇ ℝ≥0) →ₗ[ℝ≥0] ℝ≥0) (K : compacts X) :
  ⇑(riesz_content Λ) = Inf (Λ '' {f : X →ᵇ ℝ≥0 | ∀ x ∈ K, (1 : ℝ≥0) ≤ f x}) := rfl
```
because we do not want to expose `riesz_content_aux`.